packages/website/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Extends the root `.eslintrc.js`, you should just open the root directory as a project.